src/components/map/layers/popups/SelectNetworkEntityPopup.tsx

```diff
@@ -59,14 +59,25 @@ class SelectNetworkEntityPopup extends Component<ISelectNetworkEntityPopupProps>
         this.props.highlightEntityStore!.setLinks(links);
     };
 
-    private redirectToLink = (link: ILinkMapHighlight, popupId: number) => {
-        this.props.popupStore!.closePopup(popupId);
-        this.props.highlightEntityStore!.setLinks([]);
-        const linkViewLink = routeBuilder
-            .to(SubSites.link)
-            .toTarget(':id', [link.startNodeId, link.endNodeId, link.transitType].join(','))
-            .toLink();
-        navigator.goTo(linkViewLink);
+    private promptRedirectToLink = (link: ILinkMapHighlight, popupId: number) => {
```

Comment:
Maybe this should be more generic function? Especially if ILine is added to this file eventually. If not then I suppose it's not worth the hassle.